cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/SystemArguments.java

```diff
@@ -543,6 +545,21 @@ public static Map<String, String> getProfileProperties(Map<String, String> args)
     return properties;
   }
 
+  /**
+   * Get runtime arguments during app spec regeneration if skip macro evaluation flag is true
+   * @param runtimeArguments
+   * @return
+   */
+  public static Map<String, String> getRuntimeArgumentsForAppSpecRegeneration(
```

Comment:
normally the util class returns the argument related to the config, I think it is better to have this as 
```
public static boolean skipNormalMacroEvaluation()
```

Reply:
ack